x/payments/spec/05_events.md

```diff
@@ -0,0 +1,60 @@
+# Events
+
+The Payment module emits the following events:
+
+## EndBlocker
+
+| Type                            | Attribute Key            | Attribute Value       |
+|---------------------------------|--------------------------|-----------------------|
+| payment_contract_authorisation  | payer-did                | {payer-did}           |
+| payment_contract_authorisation  | payment_contract_id      | {payment_contract_id} |
+| payment_contract_authorisation  | authorised               | {authorised}          |
```

Comment:
Payment contract authorisation should be under `MsgSetPaymentContractAuthorisation`

Reply:
created